src/fhirtypes/StructureDefinition.ts

```diff
@@ -89,7 +90,7 @@ export class StructureDefinition {
    * Get the Structure Definition for Structure Definition
    * @param {ResolveFn} resolve - A function that can resolve a type to a StructureDefinition instance
    */
-  private getSdStructureDefinition(resolve: ResolveFn = () => undefined) {
+  getOwnStructureDefinition(resolve: ResolveFn = () => undefined) {
```

Comment:
Can you add a return type annotation so it's clear to users (instead of inferred)?  This also ensures if you ever change the function, you don't unintentionally change the return type.